### ISSUE TITLE
[Feature Request] Option to continue to target weakest letter after all keys are unlocked

### ISSUE THREAD

**Issue by saltedlolly** [aradzie/keybr.com#185]
Once you have unlocked all the letters at the current WPM, there could be a setting to have the algorithm continue to target whichever letter is currently weakest, only now using all letters from the alphabet, not simply a subset, regardless of which letter is currently being targetted.

So for example, once you have unlocked letter 'J' and got it to the target WPM (e.g. 35), the algorithm will switch to targeting whichever letter is currently weakest - lets say it is currently 'P'. It will then give you words from the full alphabet that all use the letter P. This differs from the previous time you attempted 'P' when it only included words with letters to the left of it. Given that you have already unlocked all letters at the target WPM it can be assumed that you know the location of every letter on the keyboard at this point without looking, so there is little need to use words with a subset of letters. It is actually better to keep using words containing all letters so you can continue to practice them, while still focussing on your weakest key. Once you have raised the target WPM on 'P' above your next weakest letter, the algorithm will then switch to targeting that letter. This should help us to continue to improve speed and accuracy on our weakest letters, while still practising all the other letters in the alphabet. As you continue to practice, always targeting your weakest letter, the target WPM could then automatically be increased as the weakest letter improved. This is perhaps a better idea than my suggestion in #179. 

With this method continuing to use Keybr, even after all letters have been unlocked, will continue to improve your typing by always targeting your weakest letter, while at the same time continuing to practice all the other letters.

**Comment by bllngr** [comment]
Yes please! I was under the impression that this is the way it works once all letters are unlocked. Was a bit disappointed to see that it basically turns into monkeytype at that point. The targeting of special letters is the unique selling point of keybr. Please let us continue to train like that even when all keys are green!
